modules/forced-mfa-users/class-forced-mfa-users.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace Automattic\VIP\Security\MFAUsers;
+
+class Forced_MFA_Users {
+	const MFA_SKIP_USER_IDS_OPTION_KEY = 'vip_security_mfa_skip_user_ids';
+
+	/**
+	 * The capability required to force MFA.
+	 *
+	 * @var string|array The capability slug or an array of slugs.
+	 */
+	private static $capability;
+
+	public static function init() {
+		if ( ! defined( 'VIP_SECURITY_BOOST_CONFIGS' ) ) {
+			return;
+		}
+	
+		$configs            = constant( 'VIP_SECURITY_BOOST_CONFIGS' );
+		$module_configs     = $configs['module_configs'] ?? [];
+		$forced_mfa_configs = $module_configs['forced-mfa-users'] ?? [];
+
+		self::$capability = $forced_mfa_configs['capability'] ?? [];
+		add_action( 'set_current_user', [ __CLASS__, 'filter_user_capabilities' ] );
+	}
+
+	/**
+	* Require 2FA based on capabilities set in config
+	*/
+	public static function filter_user_capabilities() {
+		$required_capability_or_caps = self::$capability;
+
+		if ( empty( $required_capability_or_caps ) ) {
+			return;
+		}
+
+		$user_has_required_capability = false;
```

Comment:
Similar to https://github.com/Automattic/vip-security-boost/pull/10/files#r2047021334
```suggestion
		$user_has_mfa_required = false;
```